modules/core/src/components/declarative-ui/xviz-metric.js

```diff
@@ -127,7 +127,7 @@ class XVIZMetricComponent extends PureComponent {
       verticalGridLines,
       getColor
     } = this.props;
-    const isLoading = currentTime === null;
+    const isLoading = currentTime === null || currentTime === undefined;
```

Comment:
doing `currentTime == null` would give the same result.  http://www.ecma-international.org/ecma-262/6.0/#sec-abstract-equality-comparison

I suggest this just because I myself find it bizarre to have to check 2 invalid states everywhere.  Normally strict equality is preferred, but this is the one case where abstract equality is meaningful in untyped languages.

Reply:
The linter was complaining / auto changing it to `===`. I'll switch it back to `==` and add in a linter rule for it.